src/main/common/filter.c

```diff
@@ -24,13 +24,6 @@
 #include "common/maths.h"
 #include "common/utils.h"
 
-#define M_LN2_FLOAT 0.69314718055994530942f
-#define M_PI_FLOAT  3.14159265358979323846f
-
-#define BIQUAD_BANDWIDTH 1.9f     /* bandwidth in octaves */
-#define BIQUAD_Q 1.0f / sqrtf(2.0f)     /* quality factor - butterworth*/
-
-
```

Comment:
I think it is better to have these `#define`s in `filter.c` rather than `filter.h` since they are part of the implementation, not part of the interface.